app/models/tickets_competition_result.rb

```diff
@@ -68,4 +69,12 @@ def merge_inbox_results
       self.update!(status: TicketsCompetitionResult.statuses[:merged_inbox_results])
     end
   end
+
+  DEFAULT_SERIALIZE_OPTIONS = {
+    methods: %w[competition_name],
```

Comment:
This will call/fetch the `competition` association upon every serialization. Are you sure that you pre-loaded via `TicketsCompetitionResult.includes(:competition)`  in all relevant places?